decidim-blogs/spec/forms/decidim/blogs/admin/post_form_spec.rb

```diff
@@ -34,7 +37,8 @@ module Admin
           {
             "post" => {
               "title" => title,
-              "body" => body
+              "body" => body,
+              "author_id" => author_id
```

Comment:
```suggestion
              "user_group_id" => author_id
```

user_group_id is the name of parameter that PostForm is expecting

Reply:
Done: Thank you for the suggestion. I changed the parameter.